### ISSUE TITLE
Training methodology

### ISSUE THREAD

**Issue by alasla** [cialab/deepfocus#2]
Thanks for the code/paper.  This actually works quite well on my H&E images, and not so well on a more exotic staining protocol.  I'd like to additionally train the model on some of my images produced in a similar way.  

The posted code doesn't seem to include the training code.   I assume you were calling model.fit(), but do you happen to know what arguments you used?

**Comment by senaras** [comment]
Hi, 
Since I moved to another employer, I do not have access to the whole code. But as you mentioned, you will need to call model.fit..
ex:
model.fit(trainX, trainY, n_epoch=3, shuffle=True, snapshot_step=300,  snapshot_epoch=True, show_metric=True, run_id=42, validation_set=( valX,  valY)    )

please be aware that, you will need to scan with different focal lengths to obtain different focus problems for your training dataset. 

note:I would be glad if you can let me know if you train it and it works well. I can give a link to your repo to help to the community.

**Comment by alasla** [comment]
Yeah I was hoping to get a better idea what the arguments to fit() were, but I can just test some options and see how it works.  

We’re working with our scanner manufacturer to see if we can get the ability to manual perturb the focal length.  This should let me create a handful of training images of my staining at varying focal lengths, similar to what you did in the first place.  

I’ll let you know what happens.  If I get it all working I can post a fork to GitHub that will maybe make it easier for others to train on their own data.  
—
Alex Laslavic
alexlaslavic@rubedolife.com

**Comment by senaras** [comment]
Great! I am looking forward to hearing from you!
